cli/jobs/finetuning/standalone/finetuning-job-cliv2-readme.md

```diff
@@ -0,0 +1,45 @@
+# Running a Fine-Tuning Job from CLI
+
+This guide provides instructions on how to run a fine-tuning job using the Azure Machine Learning CLI v2.
+
+## Prerequisites
+
+1. **Azure CLI**: Ensure you have the Azure CLI installed. If not, you can install it from [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+2. **Azure Machine Learning CLI v2**: Install the Azure Machine Learning CLI extension v2.
+   ```bash
+   az extension add -n ml -y
+   ```
+
+3. **Azure Subscription**: Ensure you have an active Azure subscription and the necessary permissions to create and manage resources.
+
+4. **Resource Group and Workspace**: Ensure you have an Azure resource group and an Azure Machine Learning workspace. If not, you can create them using the following commands:
```

Comment:
for MaaS , please specify the regions and that it works only in those regions. Llama in WestUS3 , Phi in EastUS2